Extension/src/LanguageServer/settings.ts

```diff
@@ -350,50 +445,58 @@ export class CppSettings extends Settings {
+    public get intelliSenseCacheSize(): number | undefined { return this.getAsNumber("intelliSenseCachePath"); }
+    public get intelliSenseMemoryLimit(): number | undefined { return this.getAsNumber("intelliSenseMemoryLimit"); }
+    public get intelliSenseUpdateDelay(): number | undefined { return this.getAsNumber("intelliSenseUpdateDelay"); }
+    public get errorSquiggles(): string | undefined { return this.getAsString("errorSquiggles"); }
+    public get inactiveRegionOpacity(): number | undefined { return this.getAsNumber("inactiveRegionOpacity"); }
+    public get inactiveRegionForegroundColor(): string | undefined { return this.getAsString("inactiveRegionForegroundColor"); }
+    public get inactiveRegionBackgroundColor(): string | undefined { return this.getAsString("inactiveRegionBackgroundColor"); }
+    public get autocomplete(): string | undefined { return this.getAsString("autocomplete"); }
+    public get autocompleteAddParentheses(): boolean | undefined { return this.getAsBoolean("autocompleteAddParentheses"); }
+    public get loggingLevel(): string | undefined { return this.getAsString("loggingLevel"); }
+    public get autoAddFileAssociations(): boolean | undefined { return this.getAsBoolean("autoAddFileAssociations"); }
+    public get workspaceParsingPriority(): string | undefined { return this.getAsString("workspaceParsingPriority"); }
+    public get workspaceSymbols(): string | undefined { return this.getAsString("workspaceSymbols"); }
+    public get exclusionPolicy(): string | undefined { return this.getAsString("exclusionPolicy"); }
+    public get refactoringIncludeHeader(): string | undefined { return this.getAsString("refactoring.includeHeader"); }
+    public get simplifyStructuredComments(): boolean | undefined { return this.getAsBoolean("simplifyStructuredComments"); }
+    public get doxygenGeneratedCommentStyle(): string | undefined { return this.getAsString("doxygen.generatedStyle"); }
+    public get doxygenGenerateOnType(): boolean | undefined { return this.getAsBoolean("doxygen.generateOnType"); }
+    public get commentContinuationPatterns(): (string | CommentPattern)[] | undefined {
+        // eslint-disable-next-line no-extra-parens
+        const value = super.Section.get<(string | CommentPattern)[]>("commentContinuationPatterns");
+        if (value === undefined) { return undefined; }
+        if (isArrayOfString(value) || this.isArrayOfCommentContinuationPatterns(value)) {
+            return value;
+        }
+        return undefined;
+    }
+    public get configurationWarnings(): boolean | undefined { return this.getAsString("configurationWarnings") === "enabled"; }
+    public get preferredPathSeparator(): string | undefined { return this.getAsString("preferredPathSeparator"); }
+    public get updateChannel(): string | undefined { return this.getAsString("updateChannel"); }
+    public get vcpkg(): boolean | undefined { return this.getAsString("vcpkg") === "enabled"; }
```

Comment:
This is the wrong pattern. If the `get` has the same name as the setting, it should return the setting's value. If it does an extra check like this, it should be called `isVcpkgEnabled`.

Reply:
This would involve changing the value of vcpkg in package.json. Are we okay with making changes to package.json for this PR?

Reply:
To clarify, do you just want to change the name of the function or the setting itself? If you just mean to change the name of the function, then that won't involve changes to package.json.

Reply:
The `get` I was referring to was the name of the function, yes. Do not change package.json in this PR.